api/valor_api/schemas/evaluation.py

```diff
@@ -116,6 +135,33 @@ def _validate_parameters(cls, values):
                             raise ValueError(
                                 "`iou_thresholds_to_return` must be a subset of `iou_thresholds_to_compute`"
                             )
+            case TaskType.TEXT_GENERATION:
+                text_comparison_metrics = set(["ROUGE", "BLEU"])
```

Comment:
metric validation needed for ROUGE and BLEU

Reply:
has this been completed?